packages/frontend-2/components/viewer/dataviewer/Row.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div
+    :class="`w-full bg-foundation-2 hover:bg-blue-500/5 rounded px-1 py-1 border-l-2 text-xs ${
+      expandable && !expanded
+        ? 'border-blue-500 border-opacity-50'
+        : 'border-neutral-500 border-opacity-20'
+    } `"
+  >
+    <!-- eslint-disable-next-line vuejs-accessibility/click-events-have-key-events -->
+    <div
+      :class="`grid grid-cols-3 ${expandable ? 'cursor-pointer' : ''}`"
+      @click="handleExpand"
+    >
+      <div
+        class="col-span-1 truncate mr-1 flex items-center text-foreground-2 font-semibold"
+      >
+        <ChevronRightIcon
+          v-if="expandable"
+          :class="`w-3 ${expanded ? 'rotate-90' : ''} transition `"
+        />
+        <span class="select-all">{{ prop.key }}</span>
```

Comment:
```suggestion
      <div
        class="col-span-1 mr-1 flex items-center text-foreground-2 font-semibold"
      >
        <ChevronRightIcon
          v-if="expandable"
          :class="`w-3 ${expanded ? 'rotate-90' : ''} transition `"
        />
        <span class="select-all truncate">{{ prop.key }}</span>
```
Currently, the first column is not truncating as expected.
<img width="181" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/970b2124-bb50-492c-af1f-6ee555add35f">